contrib/kube-local/README.md

```diff
@@ -0,0 +1,393 @@
+| AUTO_REMOVE_GOLANG_DIR | If detected a existing golang dir from previous installation, remove it. (**true** or **false**)
+
+### Logging
+| Name | Description |
+| ------ | ------ |
+| LOGGING |  Enable logging (**true** or **false**) For path, see: ${LOGDIR}/${LOGFILE} |
+| LOGDIR | Directory for storing the logs. Default: **/tmp** |
+| LOGFILE | The log file
+| LOGGING_FILTER | A filter for the type of message to be displayed in the log file. Default: DEBUG INFO WARNING FAIL OK|
+
+### Operational System
+| Name | Description |
+| ------ | ------ |
+| DISTRO_NAME | The OS Distribution name. **Automatically filled in the script** via /etc/os-release
+| DISTRO_CMD_TOOL | The OS Distribuition command name. **Automatically filled in the script**. Example dnf, apt-get, etc.
+| DISTRO_PKG_MANAGER | The OS Distribution package manager. **Automatically filled in the script**. Example: dpkg, rpm, etc.
+| DISTRO_VERSION_ID | The OS Distribution version. **Automatically filled in the script** via /etc/os-release. Example: 8 (from RHEL 8)
+
+### Golang
+| Name | Description |
+| ------ | ------ |
+| GOLANG_ARCH |  Golang arch. Default: **linux-amd64**
+| GOLANG_VERSION |  Golang version. Example: **1.13.7**
+| GOLANG_TAG |  Speficy a Golang tag name
+| GOLANG_TAR_FILENAME | Golang tar file name
+| GOLANG_TAR_URL | Golang tar URL
+| GOLANG_HASH |  Algo for hash verification. Default: **sha256**
+| GOLANG_HASH_FILENAME | Golang hash file name
+| GOLANG_HASH_URL | Golang hash URL
+| GOLANG_DIR_INSTALL | Path for installing Golang. Default: **/usr/local**
+| GOLANG_DIR_PATH | Golang path. Default: **/usr/local/go**
+| GOLANG_DIR_PATH_BIN | Golang bin path. Default: **/usr/local/go/bin**
+| GOLANG_PROJECT_HOME| Golang project **home** dir. Default: **${HOME}/go**
+| GOLANG_PROJECT_HOME_SRC| Golang project **src** dir. Default: **${GOLANG_PROJECT_HOME}/go/src**
+| GOLANG_PROJECT_HOME_SRC_GITHUB| Golang project **src** for **github.com** dir. Default: **${GOLANG_PROJECT_HOME_SRC}/github.com**
+| GOLANG_PROJECT_HOME_BIN| Golang project **bin** dir. Default: **${GOLANG_PROJECT_HOME}/go/bin**"
+| GOLANG_BIN| Path to Golang binary. Default: **${GOLANG_DIR_INSTALL}/go/bin/go**
+| GO_ENV_VARS| Path where to storing Golang variables. Default (available to all users): **/etc/profile**
+
+### Runtime
+| Name | Description |
+| ------ | ------ |
+| RUNTIME_NAME| Runtime name. Default: **cri-o**
+| RUNTIME_GITHUB_URL | Runtime github URL
+| RUNTIME_OPTIONAL_PACKAGES | Optional packages for the runtime. This option is **filled automatically inside the script** as depends of distro detection.
+| RUNTIME_VERSION | Runtime version. Example: master, release-1.13, etc
+| RUNTIME_TAG | Runtime specific tag
+| RUNTIME_GOLANG_SRC_GITHUB | Runtime Golang src github dir. Default: **github.com**
+| RUNTIME_GOLANG_DIR | Runtime Golang project dir: **${GOLANG_PROJECT_HOME_SRC}/${RUNTIME_GOLANG_SRC_GITHUB}/${RUNTIME_NAME}**
```

Comment:
I think this long def, GOLANG_PROJECT_HOME... is keeping the line from wrapping and is causing the table to add a scroll bar.  This makes none of the lines in the table wrap and it's annoying to have to scroll.  I'm not sure if that can be avoided somehow or not.  I like the thought of a table, it's just that md sometimes is painful.  Similar situation with a number of other tables below.

Reply:
Sure Tom. Made few tricks and solved that. Thanks again.